### PR TITLE
I have added Basic on how to create class and object in python

### PR DESCRIPTION
**Pull request by Khanna14Aman**
### Describe your change

- [x] Add a concept ?
- [ ] Fix a bug or typo in an existing concept ?
- [ ] Documentation change?

### Checklist

- [x] I have read [CONTRIBUTING.md](https://github.com/dev-madhurendra/OOPs/blob/main/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one file. To ease review, please open separate PRs for separate concepts.
- [x] All new folders are placed in an existing directory.
- [x] All new files are placed inside an existing directory.
- [x] All folder names should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`Golang` is allowed but `golang` is not
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`Class.cpp` is allowed but `classe.cpp` is not
- [x] All new concepts have a URL in their comments that points to Wikipedia or another similar explanation.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`

### CODE REVIEW

**Review by Suryac72 — COMMENTED** [review]
hi @Khanna14Aman , please change the file name to Class.py

**Review by Suryac72 — APPROVED** [review]
good from my side, taking forward to @dev-madhurendra

File: C++/Classes/Class.cpp [hunk 1]
```diff
@@ -25,7 +27,8 @@ class Person {
 
     // Public member functions
     void introduce() {
-        cout << "Hello, I'm " << name << " and I'm " << age << " years old." << endl;
+        cout<<"hello world";
+        // cout << "Hello, I'm " << name << " and I'm " << age << " years old." << endl;
```
Comment (dev-madhurendra) [inline C++/Classes/Class.cpp:31]: If you added class in python why c++ code.? 
Delete this file

File: Python/Class.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+
+    1- Object is a real world entity which used to access class functionality.
+    2- Memory gets allocated only when object get created.
+'''
+
+
+# Here we define a new class named "First"
+class First:
+    # This is data member/attribute of class.
```
Comment (dev-madhurendra) [inline Python/Class.py:14]: ClassName should be real world example e.g. Vehicle, Person, Rectangle e.t.c.

**Review by dev-madhurendra — CHANGES_REQUESTED** [review]

File: Python/Class.py (hunk 2 above)
Comment (dev-madhurendra) [reply Python/Class.py:14]: Also create a Class folder and move this file to Class folder only.

**Pull request closed without merging**